### ISSUE TITLE
BUG: several tests are broken with the new python module (though it is not used)

### ISSUE THREAD

**Issue by jeankpf** [idefix-code/idefix#276]
### Describe the issue:

Standard IDEFIX tests like the MHD 2D Orszag-Tang, the MHD ShearingBox or the 2D MTI are broken as from commit 134ca6b, because of a too small time step after very few cycles (possibly as from the first one). I don't know yet what is the cause of this bug, but I will try to further investigate it in the coming days.

### Error message:

```shell
Trying to run the 2D MHD Orszag-Tang test with commit a92732f3 on my laptop results in:

TimeIntegrator:             time |            cycle |        time step | cell (updates/s) |            div B
TimeIntegrator:    3.011907e-314 |                0 |    4.094819e-314 |              N/A |     0.000000e+00
Main: WARNING! Caught an exception in TimeIntegrator.
dt = 4.5043e-314 is too small.

while it's working well with commit 10dc6bf9 (after the output with python function is allowed but before the initialisation with python function is implemented)
```

### runtime information:

134ca6b, but also true for commit a92732f which is the current commit of the official pythonBinding branch.

**Comment by glesur** [comment]
Woops, I added the access to t and dt in` datablockHost` so that it could be read/write from python, but it was lacking proper initialisation when the `datablockHost `was created. Now fixed in commit  1c29ebb.

**Comment by glesur** [comment]
Woops, I added the access to t and dt in` datablockHost` so that it could be read/write from python, but it was lacking proper initialisation when the `datablockHost `was created. Now fixed in commit  1c29ebb.
